app/views/components/_document-list.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  group_index ||= false
+  track_clicks = "data-module=track-click" if group_index
+  dimension28 = items.count.to_s
+%>
+<ol class="app-c-document-list" <%= track_clicks %>>
+  <% items.each_with_index do |link, link_index| %>
+    <li class="app-c-document-list__item">
+      <h3 class="app-c-document-list__item-title">
+        <%
+          data = {}
+          if group_index
+            data = {
```

Comment:
I've checked with Paul C and apparently the tracking is still needed but stopped working on the 9th of August. Will work with him to determine a cause then look at an alternative implementation.